tests/e2e-tests/pom.xml

```diff
@@ -28,15 +28,41 @@
    <name>End-to-End Tests</name>
 
    <properties>
-      <activemq.basedir>${project.basedir}/../../</activemq.basedir>
-      <e2ets-surefire-extra-args />
+      <activemq.basedir>${project.basedir}/../..</activemq.basedir>
+      <e2e-tests.surefire-extra-args/>
+      <e2e-tests.dockerfile>Dockerfile-centos</e2e-tests.dockerfile>
+      <e2e-tests.skipImageBuild>true</e2e-tests.skipImageBuild>
+      <e2e-tests.skipTests>true</e2e-tests.skipTests>
+      <distributionDir>${activemq.basedir}/artemis-distribution/target/apache-artemis-${project.version}-bin/apache-artemis-${project.version}</distributionDir>
+      <container-service-argline>-DContainerService.artemis-image.version=${project.version} -DContainerService.artemis-image.userid="1000"</container-service-argline>
    </properties>
 
    <dependencies>
+      <dependency>
+         <!-- this dependency is here to make sure this module is only executed
+              after the distribution is created.
+              Otherwise it will get here before the build eventually.
+              e.g if you use mvn install -T 20 -->
+         <groupId>org.apache.activemq</groupId>
+         <artifactId>apache-artemis</artifactId>
+         <version>${project.version}</version>
+         <scope>compile</scope>
+         <type>pom</type>
+         <exclusions>
+            <exclusion>
+               <groupId>org.apache.activemq</groupId>
+               <artifactId>artemis-jms-client-all</artifactId>
+            </exclusion>
+            <exclusion>
+               <groupId>org.apache.activemq</groupId>
+               <artifactId>artemis-jakarta-client-all</artifactId>
+            </exclusion>
+         </exclusions>
+      </dependency>
       <dependency>
          <groupId>org.testcontainers</groupId>
          <artifactId>testcontainers</artifactId>
-         <version>1.16.2</version>
+         <version>1.16.3</version>
```

Comment:
Should be in a property, and the version controlled in dependencyManagement at the root so it is consistent across the build and can be updated easily.

Reply:
Ok @gemmellr going to change it. Also, I will do the same for smoke-test project which has a reference for testcontainers selenium module.

Reply:
PR updated.